### PR TITLE
[.NET][WPF][HTML] Add ActionSet element support

### PR DESCRIPTION
**Pull request by almedina-ms**
Adds action set support for the .NET renderers for this work item: https://github.com/Microsoft/AdaptiveCards/issues/2438

### How verified 

**Parsing**

Added two tests
* TestObjectModelActionSetElement
* TestParseActionSetElement

**Rendering**

Test apps were executed and gave this results

* WPF
image

* HTML
image

### CODE REVIEW

File: source/dotnet/Library/AdaptiveCards/AdaptiveActionSet.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        /// <summary>
+        ///     Actions of the action set
+        /// </summary>
+        [JsonConverter(typeof(IgnoreEmptyItemsConverter<AdaptiveAction>))]
+#if !NETSTANDARD1_3
+        [XmlArray("Actions")]
+        [XmlArrayItem(ElementName = "OpenUrl", Type = typeof(AdaptiveOpenUrlAction))]
+        [XmlArrayItem(ElementName = "ShowCard", Type = typeof(AdaptiveShowCardAction))]
+        [XmlArrayItem(ElementName = "Submit", Type = typeof(AdaptiveSubmitAction))]
+        [XmlArrayItem(ElementName = "ToggleVisibility", Type = typeof(AdaptiveToggleVisibilityAction))]
```
Comment (RebeccaAnne) [inline source/dotnet/Library/AdaptiveCards/AdaptiveActionSet.cs]: I'm not entirely familiar with how this works, but does this explicit list interfere with supporting the custom action scenario in this case?

File: source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs [hunk 2]
```diff
@@ -814,5 +814,176 @@ public void BackgroundImageBackCompatWithLegacyUriType()
...
+          ""type"": ""Action.ToggleVisibility"",
+          ""targetElements"": [
+            ""test""
+          ],
+          ""title"": ""Toggle""
+        }
+      ]
+    }
+  ]
+}";
```
Comment (RebeccaAnne) [inline source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs]: [nit] Indent this to line up with the code

Comment (almedina-ms) [reply source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs]: The indentation adds spaces that make the comparisson fail

File: source/dotnet/Library/AdaptiveCards/AdaptiveActionSet.cs (hunk 1 above)
Comment (almedina-ms) [reply source/dotnet/Library/AdaptiveCards/AdaptiveActionSet.cs]: I took this part of code from the AdaptiveCard class 

just talked with Carlos, apparently there's some support for custom actions added in the falllback support (will have to check if there's actual custom action support) so when fallback PR is merged, I'll update this section

---
In reply to: [272653316](https://github.com/Microsoft/AdaptiveCards/pull/2614#discussion_r272653316) [](ancestors = 272653316)

**Comment by shalinijoshi19** [comment]
@almedina-ms any reason why this is still open ?

File: source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs (hunk 2 above)
Comment (RebeccaAnne) [reply source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs]: Makes sense. Could you add a comment to that effect?

---
In reply to: [273150076](https://github.com/Microsoft/AdaptiveCards/pull/2614#discussion_r273150076) [](ancestors = 273150076)

File: source/dotnet/Library/AdaptiveCards/AdaptiveActionSet.cs (hunk 1 above)
Comment (RebeccaAnne) [reply source/dotnet/Library/AdaptiveCards/AdaptiveActionSet.cs]: Sounds good. If that ends up being a separate change, could you log an issue so we don't lose track of it?

---
In reply to: [273178556](https://github.com/Microsoft/AdaptiveCards/pull/2614#discussion_r273178556) [](ancestors = 273178556,272653316)

Comment (almedina-ms) [reply source/dotnet/Library/AdaptiveCards/AdaptiveActionSet.cs]: https://github.com/Microsoft/AdaptiveCards/issues/2676

**Pull request merged**